server/routes/referrals/referralsController.ts

```diff
@@ -196,13 +202,8 @@ export default class ReferralsController {
     }
 
     if (!error) {
-      res.redirect(`/referrals/${req.params.id}/desired-outcomes`)
+      res.redirect(`/referrals/${req.params.id}/service-category/${referral.serviceCategoryIds[0]}/desired-outcomes`)
     } else {
-      const referral = await this.interventionsService.getDraftReferral(
-        res.locals.user.token.accessToken,
-        req.params.id
-      )
-
       if (!referral.serviceCategoryId) {
         throw new Error('Attempting to view relevant sentence without service category selected')
```

Comment:
ah, i see it matches this message lower down.

Reply:
No, you're right to nitpick - it's fine to have 2 different error messages here and clearer. Have updated this.